### ISSUE TITLE
Logout not working properly.

### ISSUE THREAD

**Issue by hilquias-kil** [auth0-samples/auth0-react-samples#100]
https://github.com/auth0-samples/auth0-react-samples/blob/4d9efbc59d3a5086ac0aeb3962d125b70c5da6d8/01-Login/src/Auth/Auth.js#L48-L55

To work properly, this method should call the `auth0.logout`:
```js
this.auth0.logout({
    returnTo: URL,
    client_id: AUTH_CONFIG.clientId
});
```

Thanks.

**Comment by matejstrasek** [comment]
This should be added to Quick start and sample project.

**Comment by pejowei** [comment]
Having followed this [React Quickstart](https://auth0.com/docs/quickstart/spa/react) tutorial to the letter, I can confirm the provided logout is _not_ working correctly.

Using the tutorial's method, if a user explicitly logs out, and then invokes the login method again, the user is logged back into the app without a prompt for username and password.

Thanks @hilquias-kil , this logout function works for me.   To anyone else trying this, make sure you add your LOGOUT_URL to the "Allowed Logout URLs" on the Auth0 application management dashboard.

`
public logout = () => {
        // Remove tokens and expiry time
        this.accessToken = null;
        this.idToken = null;
        this.expiresAt = 0;

        // Remove isLoggedIn flag from localStorage
        localStorage.removeItem('isLoggedIn');

        // navigate to the home route
        this.auth0.logout({
            returnTo: LOGOUT_URL,
            clientID: CLIENT_ID}
            )
`

**Comment by albertoperdomo** [comment]
Hello,

thanks for taking the time to report this. This is addressed in #108.
